### PR TITLE
Added some quality badges to the README

### PR DESCRIPTION
**Pull request by rascaraficci**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
NA

* **What is the current behavior?** (You can also link to an open issue here)
NA

* **What is the new behavior (if this is a feature change)?**
NA

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
NA

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
No

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# microservice-sdk
+# microservice-sdk [![DeepScan grade](https://deepscan.io/api/teams/2690/projects/13298/branches/219450/badge/grade.svg)](https://deepscan.io/dashboard#view=project&tid=2690&pid=13298&bid=219450) [![CodeFactor](https://www.codefactor.io/repository/github/dojot/dojot-microservice-sdk-js/badge)](https://www.codefactor.io/repository/github/dojot/dojot-microservice-sdk-js) [![codecov](https://codecov.io/gh/dojot/dojot-microservice-sdk-js/branch/master/graph/badge.svg)](https://codecov.io/gh/dojot/dojot-microservice-sdk-js)
```
Comment (eduardogmisiuk) [inline README.md:1]: Isn't it better `Microservice SDK` instead of `microservice-sdk`?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 
 This nodejs module aims to help developers to build microservices in the dojot's context.
```
Comment (eduardogmisiuk) [inline README.md:3]: NodeJS

File: README.md (hunk 1 above)
Comment (rascaraficci) [reply README.md:1]: I agree with you!

File: README.md (hunk 2 above)
Comment (rascaraficci) [reply README.md:3]: :+1:

**Pull request merged**